### ISSUE TITLE
"This block has encountered an error and cannot be previewed" when adding link

### ISSUE THREAD

**Issue by gpetty** [WordPress/gutenberg#14962]
**Describe the bug**
I had previously worked some months ago with Gutenberg with no problem. Today, however, I frequently encounter the error "This block has encountered an error and cannot be previewed." I cannot recover the block but must delete it and start fresh.  This has always happened when I was trying to insert a link into a paragraph block -- e.g., highlight text, Command-K, Command-V to paste URL.  However, often the error does not occur.
The error was first encountered in v4 of Gutenberg but still occurs after updating to 5.4.0.  I also updated Wordpress before editing today but don't know whether that is related.

**To reproduce**
1. Highlight text in a paragraph block
2. Command-K (on Mac) to open link editor
3. Command-V to paste URL
4. <return> to accept change
5. Sometimes the error occurs, sometimes it does not

**Expected behavior**
Ability to create links in a paragraph block without generating an error that requires the entire block to be deleted.

**Screenshots**

**Desktop (please complete the following information):**
 - OS:  Mac OS X 10.13.6
 - Browser: Firefox 
 - Version 66.0.2

**Additional context**
- Please add the version of Gutenberg you are using in the description.
- To report a security issue, please visit the WordPress HackerOne program: https://hackerone.com/wordpress.

**Comment by nerrad** [comment]
Thanks for reporting this issue.  To rule out this being a potential issue with other plugins/themes that might be creating conflicts with the editor you can try temporarily deactivating your plugins and use a different theme.  Then see if the issue still persists.  If it doesn't then it will need reported to whatever plugin/theme is causing the conflict.

**Comment by jorgefilipecosta** [comment]
Hi @gpetty,  I was not able to reproduce this issue. If possible follow the suggestion by @nerrad if even in a test website without plugins/non-default theme the problem still happens feel free to reopen the issue and we will look further.

**Comment by vesaraiskila** [comment]
I just encountered this issue. All the content of a block just disappeared with this cryptic error message.

**Comment by bradymills** [comment]
same here

**Comment by gbassoc** [comment]
Since I upgraded to WP 5.2 I started getting the same error message and the editor width is now about 1/4 the size  it used to be before I upgraded to 5.2. Existing blocks are still full size in the editor, it only new blocks that have this size issue, not only that, there are now odd informational text messages I never saw before. 
I have attached a screenshot.
I have done the usual troubleshooting, i.e. removing all the plugins and only using the default there but  that actions did not resolve the issue with Gutenberg editor. 
![gutenberg-editor-issue-2019 05 11-23-28-54](https://user-images.githubusercontent.com/10323770/57575740-97842d00-7448-11e9-8534-afebfdc80afe.png)

In the second screenshot you will also notice the style of the for the buttons in the block search popup window have also changed.
![gutenberg-editor-issue-2](https://user-images.githubusercontent.com/10323770/57575786-504a6c00-7449-11e9-9027-146b79524e43.png)

**Comment by nerrad** [comment]
To those posting to this issue, please provide as much detail as you can along with any screenshots.  This includes things like:

- What browser and browser version you are using,
- What operating system you are using,
- What version of Gutenberg you are running (i.e. what is bundled with WordPress, or the plugin and what version)
- What version of WordPress you are running.

Also, please be as specific as possible about the steps you take to reproduce this problem - and indicate whether it sporadically happens with those steps or consistently.  Being specific also can include the _type_ of content you are adding (for example, are you pasting from another source such as Google Docs?)

All this information is super helpful in aiding with finding the source of the issue and ultimately fixing it (if it can be reproduced).  Don't assume those reading your post or viewing any screenshots will understand what is happening in your environment.  

While some posting in this thread have indicated they've deactivated any plugins and are using a default theme (such as twenty nineteen), it is very important for anyone posting additional info do the same so we can eliminate the cause being a conflict with another plugin or them.

Thanks!

**Comment by gbassoc** [comment]
HI Darren
After I added the comments I realised this thread was closed. I have opened a new thread https://github.com/WordPress/gutenberg/issues/15586 with all the information you have requested

**Comment by Wnfdiary** [comment]
I have the same problem, when i try to add link. I tested on another theme with all plugins and it works normal. So the problem is with the theme - lets blog theme by themegoods . I got this error after I installed WordPress 5.2.2. The same error i got with spacer block. However if I change to the code editor and than back to visual editor, the block comes back. I have tried both Chrome and IE.

Thank you
